modules/cert-manager-acme-dns01-explicit-aws.adoc

```diff
@@ -3,24 +3,20 @@
 // * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
 
 :_content-type: PROCEDURE
-[id="cert-manager-acme-dns01-aws_{context}"]
-= Configuring an ACME issuer to solve DNS-01 challenges
+[id="cert-manager-acme-dns01-explicit-aws_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using explicit credentials for AWS Route53
 
-You can use {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Amazon Route 53.
-
-[NOTE]
-====
-Private DNS zones are not supported.
-====
+You can use {cert-manager-operator} to set up an Automated Certificate Management Environment (ACME) issuer to solve DNS-01 challenges by using explicit credentials on AWS. This procedure uses _Let's Encrypt_ as the ACME certificate authority (CA) server and shows how to solve DNS-01 challenges with Amazon Route 53.
 
 .Prerequisites
 
-* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+
+* You have access to a role that has permissions to create issuers, certificates, and secrets in the desired namespace.
```

Comment:
There are four occurrences of `You have access to a role that has permissions to create issuers ...`. This is not needed. This is by default for users. All four occurrences can be removed.

Reply:
+1 to cleaning things up from duplication, but please take care of the pre-requisites in a feasible manner so users are aware what is the least permissions needed for carrying out the steps on a cluster.